libvirt.go

```
@@ -318,7 +318,10 @@ func (d *Driver) Start() error {
 
 	for i := 0; i < 90; i++ {
 		time.Sleep(time.Second)
-		ip, _ := d.GetIP()
+		ip, err := d.GetIP()
+		if err != nil {
+			return fmt.Errorf("%v: getting ip during machine start", err)
+		}
 		if ip != "" {
 			// Add a second to let things settle
```

I think this loop is ignoring errors on purpose, and is busy looping while it waits for the VM to start. Do you have a specific error in mind when we could return early because we know we are never recover from that error?

---

Still the same comment, I believe the whole point of the loop is to wait until everything is up. If you are reasonably sure that we should be looping while we get `"", nil` from `GetIP()`, but that if we get `"", err` from `GetIP()`, then this is never going to be a transient error (ie the error won't go away once the boot process has progressed enough), then please explain this in the commit log.

---

No input on this one?